markets/perps-market/contracts/modules/PerpsAccountModule.sol

```diff
@@ -85,4 +87,20 @@ contract PerpsAccountModule is IAccountModule {
         );
         return (pnl, accruedFunding, position.size);
     }
+
+    function submittedAsyncOrder(
+        uint128 accountId,
+        uint128 marketId
+    ) external view override returns (int) {
+        PerpsMarket.Data storage perpsMarket = PerpsMarket.loadValid(marketId);
```

Comment:
optional check; could remove

Reply:
need to load it either (at least until we change the location of asynchOrderClaims